src/test/resources/application.properties

```diff
@@ -1,4 +1,15 @@
 spring.datasource.driver-class-name=org.h2.Driver
 spring.datasource.url=jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
 spring.datasource.username=sa
-spring.datasource.password=sa
\ No newline at end of file
+spring.datasource.password=sa
+
+#spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_project?serverTimezone=Europe/Warsaw&useSSL=False
+#spring.datasource.username=kodilla_user
+#spring.datasource.password=kodilla_password
+
+
+#spring.jpa.database=mysql
+#spring.jpa.show-sql=true
+#spring.jpa.hibernate.ddl-auto=update
+#spring.jpa.hibernate.naming.strategy=org.hibernate.cfg.ImprovedNamingStrategy
+#spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
```

Comment:
Raczej nie jest to potrzebne

Reply:
przydaje się jak chchemy przeprowadzić testy dodatkowo w bazie mySQL ja je robiłem i będe jeszcze robił w dalszych testach dlatego zostawiam

Reply:
już testy poprawion przechodzą Circleci

Reply:
https://circleci.com/gh/klikuskodilla/JDP-1912-02/73?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link